### ISSUE TITLE
Question about zeroize/none operation

### ISSUE THREAD

**Issue by LicharYuan** [D-X-Y/NATS-Bench#19]
**Describe the Question**
A clear and concise description of the question.
- Is it about the topology search space in NATS-Bench? Y
- Is it about the size search space in NATS-Bench? N
- Which figure or table are you referring to in the paper? N

Hi, I am trying to use [2] to search in nats-bench(NATS) [1], but there are some differences in the model's presentations between NATS and [2]. So I have to do some transfer, but I  still confuse about NATS presentation. 

Let us check this arch:

```python
In [12]: tss.query_info_str_by_arch(2211)
[2021-06-24 04:52:46] Call query_info_str_by_arch with arch=2211and hp=12
[2021-06-24 04:52:46] Call query_index_by_arch with arch=2211
[2021-06-24 04:52:46] Call clear_params with archive_root=../NATS-tss-v1_0-3ffb9-simple and index=2211
Out[12]: "|none~0|+|none~0|nor_conv_3x3~1|+|skip_connect~0|skip_connect~1|nor_conv_1x1~2|\ndatasets : ['cifar10-valid', 'cifar10', 'cifar100', 'ImageNet16-120'], extra-info : arch-index=2211\ncifar10-valid  FLOP= 47.10 M, Params=0.344 MB, latency=15.46 ms.\ncifar10-valid  train : [loss = 0.710 & top1 = 75.25%], valid : [loss = 0.758 & top1 = 73.74%]\ncifar10        FLOP= 47.10 M, Params=0.344 MB, latency=15.46 ms.\ncifar10        train : [loss = 0.577 & top1 = 80.37%], test  : [loss = 0.601 & top1 = 78.78%]\ncifar100       FLOP= 47.11 M, Params=0.350 MB, latency=14.45 ms.\ncifar100       train : [loss = 2.019 & top1 = 46.12%], valid : [loss = 2.052 & top1 = 46.16%], test : [loss = 2.058 & top1 = 44.88%]\nImageNet16-120 FLOP= 11.78 M, Params=0.351 MB, latency=14.66 ms.\nImageNet16-120 train : [loss = 3.214 & top1 = 22.78%], valid : [loss = 3.158 & top1 = 24.83%], test : [loss = 3.216 & top1 = 22.53%]"
```

I assume `none~` means  zeroize in Fig.1[1], In this model, node0->node1 w/o connection, is that means node1->node2 connection failed even with operations presentation (`nor_conv_3x3` in this case), same as node1->node3?  If so, the cell is identity cell, w/o any connection?

Refer:
[1] NATS-Bench: https://arxiv.org/pdf/2009.00437.pdf
[2] Neural Predictor for Neural Architecture Search: https://arxiv.org/pdf/1912.00848.pdf

**Comment by D-X-Y** [comment]
Sorry for the late reply. `none` refers to the zero operation.
In this case: `|none~0|+|none~0|nor_conv_3x3~1|+|skip_connect~0|skip_connect~1|nor_conv_1x1~2|`
- node-1: zero
- node-2: zero + conv(zero) = a constant bias term
- node-3: node-0 + zero + conv(node-2)

It is similar to identity cell but has a little bit difference.

**Comment by LicharYuan** [comment]
Gotta. The `none` OP still has a connection but set the zero mask. Thanks!

**Comment by D-X-Y** [comment]
Yes, exactly~
